travis/clone_oca_dependencies

```diff
@@ -41,20 +41,20 @@ def parse_depfile(depfile, owner='OCA'):
         if not line or line.startswith('#'):
             continue
         parts = line.split()
-        repo = parts[0]
-        if len(parts) > 2:
-            branch = parts[2]
-        else:
+        repo, url, branch, commit = [
+            len(parts) > i and parts[i] for i in range(4)]
+        if not branch:
             branch = os.environ.get('VERSION', '8.0')
-        if len(parts) > 1:
-            url = parts[1]
-        else:
+        if not url:
             url = 'https://github.com/%s/%s.git' % (owner, repo)
-        deps.append((repo, url, branch))
+        if commit and commit.startswith('#'):
+            # It's not a commit, it's an inline comment
+            commit = False
```

Comment:
@moylop260 I copied this from Fentanez's version but I don't see any documentation of inline comments in oca_dependencies.txt file, what do you think about?

Reply:
Let's to remote it since that the documentation is supporting comments at start of the line